include/rmm/mr/device/binning_memory_resource.hpp

```diff
@@ -151,11 +151,11 @@ class binning_memory_resource final : public device_memory_resource {
    * @param bytes Requested allocation size in bytes
    * @return rmm::mr::device_memory_resource& memory_resource that can allocate the requested size.
```

Comment:
*nit*: `rmm::async_resource_ref`? And maybe change the `@brief` to "Get the resource reference for the requested size"?